go.mod

```diff
@@ -190,6 +190,8 @@ require (
 	github.com/prometheus/client_golang v1.14.0
 	github.com/prometheus/common v0.37.0
 	github.com/rboyer/safeio v0.2.1
+	github.com/robfig/cron v1.2.0
```

Comment:
does `go mod tidy` remove this?

Reply:
Just noticed the import. We want to import `"github.com/robfig/cron/v3"` and then go mod tidy should clean this up